infrastructure/k8s/nats.tf

```diff
@@ -0,0 +1,250 @@
+resource "kubernetes_namespace" "nats" {
```

Comment:
Probably something more than just `nats` for the name.

Reply:
it's actually short for `Neural Autonomic Transport System` which is cool but a mouthful, pretty much everyone calls it `nats`, we could use `event-bus` for the namespace, that's accurate without being hard to remember